src/components/DateRangePicker.res

```diff
@@ -540,7 +540,9 @@ module Base = {
     }
 
     let buttonText = {
-      showTime
+      startDateVal->String.length <= 0 && endDateVal->String.length <= 0
```

Comment:
We can directly write `startDateVal->String.length === 0 && endDateVal->String.length === 0`